specutils/manipulation/extract_spectral_region.py

```diff
@@ -194,3 +196,64 @@ def spectral_slab(spectrum, lower, upper):
     region = SpectralRegion(lower, upper)
 
     return extract_region(spectrum, region)
+
+
+def extract_bounding_spectral_region(spectrum, region):
+    """
+    Extract the entire bounding region that encompasses all sub-regions
+    contained in a multi-sub-region instance of `~specutils.SpectralRegion`.
+
+    In case only one sub-region exists, this method is equivalent to
+    `extract_region`.
+
+    Parameters
+    ----------
+    spectrum: `~specutils.Spectrum1D`
+        The spectrum object from which the region will be extracted.
+
+    region: `~specutils.SpectralRegion`
+        The spectral region to extract from the original spectrum, comprised
+        of one or more sub-regions.
+
+    Returns
+    -------
+    spectrum: `~specutils.Spectrum1D`
+        Excised spectrum from the bounding region defined by the set of
+        sub-regions in the input `region` instance.
```

Comment:
This should fix the docs failure in the PR
```suggestion
        sub-regions in the input ``region`` instance.
```